gen/hooks.py

```diff
@@ -99,6 +99,10 @@ def function_hook(fn, data):
     # Python exposed function name conveerted to camelcase
     x_name = m.group(1)
     x_name = x_name[0].lower() + x_name[1:]
+    
+    # Make create functions less visible
+    if x_name.startswith('create'):
+        x_name = '_' + x_name
```

Comment:
There's actually a bunch of different functions that are internal, like the versioned motion profiling functions. I think this should live in the autogen YAML configs.

Reply:
Maybe an 'internal: true' flag in the YAML then?